docs/console-winevents.md

```diff
@@ -52,7 +52,7 @@ The following event constants are used in the *event* parameter of the [*WinEven
 <strong>EVENT_CONSOLE_CARET</strong>
 0x4001</td>
 <td><p>The console caret has moved. The <em>idObject</em> parameter is one or more of the following values: <strong>CONSOLE_CARET_SELECTION</strong> or <strong>CONSOLE_CARET_VISIBLE</strong>.</p>
-<p>The <em>idChild</em> parameter is a [<strong>COORD</strong>](coord-str.md) structure that specifies the cursor's current position.</p></td>
+<p>The <em>idChild</em> parameter is a <a href="coord-str.md" data-raw-source="[&lt;strong&gt;COORD&lt;/strong&gt;](coord-str.md)"><strong>COORD</strong></a> structure that specifies the cursor&#39;s current position.</p></td>
```

Comment:
Is this the correct way to format links to MD files?

Reply:
Markdig table formatting: https://review.docs.microsoft.com/en-us/new-hope/engineering/tools/markdig/markdig-migration-information?branch=master&tabs=dfm

Reply:
Thanks Matt, but this is how our links were transformed during the conversion to MarkDig … and they looked funny which is why I asked here ;)

Ultimately, it'd be great if we could drop the `data-raw-source` field since that's of relatively little use since MD links aren't permitted within an HTML element.

Also, shouldn't the <a href="..."> address reference the transformed .html file, not the .md file?

Reply:
Ahh - I see... they probably have the converter just switch it to HTML to keep the format consistent. Removed that data-raw-source garbage. 
The page is still a markdown file, just with embedded HTML... better to leave it as .md to be consistent and so that the YAML block works fine... and maybe if there's a reason for someone at some point they can convert the table from HTML to Markdig format.